apps/server/src/modules/lti-tool/lti-tool.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { LtiToolRepo } from '@shared/repo';
+import { LegacyLogger } from '@src/core/logger';
+import { LtiToolService } from './service';
+
+@Module({
+	providers: [LtiToolService, LtiToolRepo, LegacyLogger],
```

Comment:
I know you didnt introduced the ltiToolRepo but how many services uses it? Only idTokenService? Maybe you can move it this module under /repo

Reply:
nextcloud adapter also uses it